.gitignore

```diff
@@ -0,0 +1,41 @@
+# Local .terraform directories
+**/.terraform/*
+
+# .tfstate files
+*.tfstate
+*.tfstate.*
+
+# Crash log files
+crash.log
+crash.*.log
+
+# Exclude all .tfvars files, which are likely to contain sensitive data, such as
+# password, private keys, and other secrets. These should not be part of version 
+# control as they are data points which are potentially sensitive and subject 
+# to change depending on the environment.
+*.tfvars
```

Comment:
@yahya130 we've got explicit entries for certain tfvars files at the bottom of this file. Generally, we shouldn't include tfvars files since they often contain secrets or environment specific stuff unless explicitly included (i.e. - dev.tfvars we know won't contain secrets). 

Does that address your concern?